lib/linguist/languages.yml

```diff
@@ -5088,6 +5080,25 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#76d275"
+  extensions:
+  - ".bazel"
+  - ".bzl"
+  - ".star"
```

Comment:
I'm a bit wary of adding that file extension. There are lots of a `.star` data files and they will all be recognized as Starlark with this pull request as-is.

Reply:
Yeah, I think you’re right. While samples I’ve found are either Starlark or a C code with the `.star` as an extension (and further usages that don’t make much sense) — the `.star` extension is not mentioned anywhere as the official Starlark one. Removing.